docs/src/components/reference/rehypeReference.mjs

```diff
@@ -88,7 +88,10 @@ export function rehypeReference() {
             subtree.push(
               createMdxElement({
                 name: 'h3',
-                children: [{ type: 'text', value: name }],
+                children: [{ type: 'text', value: name.replace(/\s/g, '') }],
+                props: {
+                  id: name.toLowerCase().replace(/\s/g, '-'),
+                },
```

Comment:
Check `startCase()` just above where the original conversion from PascalCase happens. We can preserve the original name and just use `kebabCase()` to create a hyphenated anchor id